### DISCUSSION TITLE
Identify SSD Temperatures?

### DISCUSSION THREAD

**Discussion by JazzyJazzjazzy** [T-Troll/alienfx-tools#327] [General]
I used .msi to install AlienFx in order to get detection and listing of sensors, instead of generic ESIF Sensor label. This was to properly install startup.dvx and remoteui.dvx.

I cannot determine which is my two SSD temperatures. One is Samsung and one is Western Digital. Already used Samsung Magician (SM) and Western Digital Dashboard (WDD) but their temperatures differ by 5 to 14 degrees from what is displayed in AlienFan so I cannot "match" them up.

Can you suggest another method to identify my SSDs temperatures? Also, registry lists several SSD items and I only have two installed. There are three SSD slots. Thank you. 

![Temps (Phone)](https://user-images.githubusercontent.com/40358464/227961202-4af6abe8-f82f-4996-af55-389c2d9c718e.png)

**Comment by T-Troll** [comment]
Interesting... I got SSD temperatures via WMI call (Windows collect SMART data there). But i see it's not available in your case, this is strange.
Oh... I check and see i lost it too! Bug or MS remove it in latest update?

**Reply by T-Troll** [reply]
UPD: I check, yes, MS change something. I still can get SMART, but "Temperature" field not available.
UPD2: Found what happened, will be fixed into next release.

**Reply by JazzyJazzjazzy** [reply]
Newest release 8.2.6.2 populates SENSOR field with 11 entries. Previously, I had only nine entries. How can I check that all 11 entries  are valid? Kindly note  that SSD Temps are back. Thank you.

**Reply by T-Troll** [reply]
Well... All 11 sensors PRESENT into the system, definitely. Also, all should have AWCC/ESIF names.
But i forced to change detection slightly, so some of them can have no reading (like always 0 or -1) - just ignore them.
